src/crush/CrushTester.cc

```diff
@@ -364,6 +364,7 @@ int CrushTester::test_with_crushtool(const string& crushtool,
 {
   string timeout_string = stringify(timeout);
   string opt_max_id = stringify(max_id);
+  string opt_ruleset = stringify(ruleset);
```

Comment:
@choury You can change the title of the pull request if you like, but please do not touch the commit message. This is a backport, so the commit message is derived from the master commit, with any changes described in the Conflicts section. And @choury you did this very well already - your commit message is excellent.

@amitkumar50 Please read our backporting guidelines at http://tracker.ceph.com/projects/ceph-releases/wiki - I believe your reviews are well-intentioned, but for backports we want as few changes (compared to master) as possible. This is explained in the guidelines.

Bottom line: all changes must go into master first, if possible, because master is where development takes place.

OK?